client/navigation-container.js

```diff
@@ -28,67 +29,89 @@ const NavigationContainer = ( { menuItems } ) => {
 
+	// Make sure "Home" has a title. This should be fixed in wc-admin
+	// See https://github.com/woocommerce/woocommerce-admin/pull/5108/files#r486751290
+	const home = menuItems.find( ( items ) => items.id === 'woocommerce-home' );
+	home.title = 'Home';
+
+	// Analytics is a category and should not have a url. Temporary fix.
+	// See https://github.com/WordPress/gutenberg/issues/25251
+	const analytics = menuItems.find(
+		( items ) => items.id === 'woocommerce-analytics'
+	);
+	analytics.url = undefined;
+
+	const categories = menuItems.filter( ( item ) => item.isCategory );
+
+	const [ activeItem, setActiveItem ] = useState( 'woocommerce-home' );
+	const [ activeLevel, setActiveLevel ] = useState( 'woocommerce' );
+
 	return (
 		<div className="woocommerce-navigation">
 			<Navigation
-				activeItemId={ 'active' }
-				data={ menuItems }
-				rootTitle="Home"
+				activeItem={ activeItem }
+				activeMenu={ activeLevel }
+				onActivateItem={ setActiveItem }
+				onActivateMenu={ setActiveLevel }
 			>
-				{ ( { level, parentLevel, NavigationBackButton } ) => {
-					return (
-						<>
-							{ parentLevel && (
-								<NavigationBackButton>
-									<Icon icon={ arrowLeft } />
-									{ parentLevel.title }
-								</NavigationBackButton>
-							) }
-							{ ! parentLevel && dashboardUrl && (
-								<Button
-									isPrimary
-									href={ dashboardUrl }
-									className="woocommerce-navigation__back-button"
-								>
-									<Icon icon={ arrowLeft } />
-									{ __(
-										'WordPress Dashboard',
-										'woocommerce-navigation'
-									) }
-								</Button>
-							) }
-							<h1>{ level.title }</h1>
-							<NavigationMenu>
-								{ level.children
-									.filter(
-										( item ) => item.menuId !== 'secondary'
-									)
-									.map( ( item ) => {
-										return (
-											<NavigationMenuItem
-												{ ...item }
-												key={ item.id }
-											/>
-										);
-									} ) }
-							</NavigationMenu>
-							<NavigationMenu>
-								{ level.children
-									.filter(
-										( item ) => item.menuId === 'secondary'
-									)
-									.map( ( item ) => {
-										return (
-											<NavigationMenuItem
-												{ ...item }
-												key={ item.id }
-											/>
-										);
-									} ) }
-							</NavigationMenu>
-						</>
-					);
-				} }
+				{ activeLevel === 'woocommerce' && dashboardUrl && (
+					<Button
+						isPrimary
+						href={ dashboardUrl }
+						className="woocommerce-navigation__back-button"
+					>
+						<Icon icon={ arrowLeft } />
+						{ __( 'Dashboard', 'woocommerce-navigation' ) }
+					</Button>
+				) }
+				{ categories.map( ( category ) => (
+					<NavigationMenu
+						key={ category.id }
+						title={ category.title }
+						menu={ category.id }
+						parentMenu={
+							activeLevel !== 'woocommerce' && category.parent
+						}
+						backButtonLabel={
+							activeLevel !== 'woocommerce' && 'WooCommerce Home'
```

Comment:
This should be translated.
```suggestion
							activeLevel !== 'woocommerce' && __( 'WooCommerce Home', 'woocommerce-navigation' )
```